[bug] Removed node breaks network graph

**Version**

Build/Run method
- [x] Docker
- [ ] PKG
- [ ] Manually built (git clone - npm install - npm run build ) 

zwavejs2mqtt version: v1.0.0-alpha.2

**Describe the bug**
A removed node (node id 5 was excluded) breaks the network graph. Seems to be similar to the closed issue #87.

Frontend console throws this exception:

```
vue.esm.js:1897 Error: missing: 5
    at D (vue-d3-network.umd.js:5313)
    at d (vue-d3-network.umd.js:5363)
    at Function.B.u.initialize (vue-d3-network.umd.js:5394)
    at f (vue-d3-network.umd.js:5727)
    at Object.force (vue-d3-network.umd.js:5769)
    at o.simulate (vue-d3-network.umd.js:7753)
    at o.animate (vue-d3-network.umd.js:7785)
    at o.reset (vue-d3-network.umd.js:7789)
    at o.netLinks (vue-d3-network.umd.js:7633)
    at pn.run (vue.esm.js:4577)
```

**To Reproduce**
Exclude a node from the network.

**Expected behavior**
Graph should still be working.

**Additional context**
Replicated in :latest and :dev tags.

--- a/src/components/Mesh.vue
+++ b/src/components/Mesh.vue
@@ -235,14 +235,17 @@ export default {
     updateLinks () {
       this.links = []
 
-      for (const source of this.nodes) {
+      for (const source of this.activeNodes) {
         if (source.neighbors) {
           for (const target of source.neighbors) {
-            this.links.push({
-              sid: source.id,
-              tid: target,
-              _color: this.$vuetify.theme.dark ? 'white' : 'black'
-            })
+            // ensure target node exists
+            if (this.nodes[target] && this.nodes[target].status !== 'Removed') {
+              this.links.push({
+                sid: source.id,
+                tid: target,
+                _color: this.$vuetify.theme.dark ? 'white' : 'black'
+              })
+            }
           }
         }
       }

